jest/sfccPathSetup.js

```diff
@@ -190,18 +206,18 @@ jest.mock('*/cartridge/adyenConstants/paymentMethodDescriptions', () => {
   return require('../src/cartridges/int_adyen_overlay/cartridge/adyenConstants/paymentMethodDescriptions');
 }, {virtual: true});
 
-jest.mock('*/cartridge/controllers/middlewares/adyen/checkBalance', () => {
-  return require('../src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/checkBalance');
+jest.mock('*/cartridge/scripts/util/riskDataHelper', () => {
```

Comment:
These entries up until the end of the file are new. The things that git thinks it has replaced are actually moved up (see line 36 to 51)